cljfmt/src/cljfmt/core.cljc

```diff
@@ -568,10 +568,16 @@
 #?(:clj
    (defn- as-zloc->alias-mapping [as-zloc]
      (let [alias             (some-> as-zloc z/right z/sexpr)
-           current-namespace (some-> as-zloc z/leftmost z/sexpr)
+           current-namespace (some-> as-zloc
+                                     z/leftmost
+                                     (z/find #(n/symbol-node?
+                                               (z/node (skip-meta %))))
+                                     z/sexpr)
```

Comment:
If you have something like 

```clj
[#_my.old.namespace my.namespace :as n]
```

then `z/leftmost` will be an `:uneval` node (see https://github.com/clj-commons/rewrite-clj/issues/70) so we need to skip it until we find the first **symbol** node, hence the use of `z/find`. I didn't use `token?` here since keywords are tokens too and we don't want it to accidentally return `:as` if it runs into some other issue.